detection_rules/rule.py

```diff
@@ -185,7 +185,7 @@ def validate(self, as_rule=False, versioned=False, query=True):
 
         schema_cls.validate(contents, role=self.type)
 
-        skip_query_validation = self.metadata['maturity'] == 'development' and \
+        skip_query_validation = self.metadata['maturity'] in ('beta', 'development') and \
```

Comment:
I think we need a new `experimental` maturity level

Reply:
Was thinking the same thing 👍